tests/testthat/test_normalize.R

```diff
@@ -0,0 +1,6 @@
+test_that("function: all normalize methods", {
+    for (.method in normalizeMethods()) {
```

Comment:
Are you planning to add some more unit tests here, e.g. to check that the methods actually do what they should?

Reply:
This could be done, indeed. I am not sure to what extend is will be easy to do though, in some cases at least. Such unit tests should probably be added upstreams... 
But for now, I wanted to make sure all functions could be called.

Reply:
sure, totally understand - also, by looking at the code it is pretty straight forward and one might expect the functions to work properly...